core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -70,6 +71,43 @@
       "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
   private static final Random RANDOM = new Random();
 
+  /**
+   * Convenience method for calling {@link #createProgressThread(long, PrintStream)} with an
+   * interval of 2 seconds.
+   *
+   * @param stream the print stream to write to
+   * @return the thread
+   */
+  public static Thread createProgressThread(PrintStream stream) {
+    return createProgressThread(2 * Constants.SECOND_MS, stream);
+  }
+
+  /**
+   * Creates a thread which will write "." to the given print stream at the given interval. The
+   * created thread is not started by this method. The created thread will be daemonic and will
```

Comment:
nit: daemonic=demonic="A persistently tormenting person"

Reply:
Change to a daemon thread, Thanks for explanation